App/res/values/strings.xml

```diff
@@ -323,6 +324,7 @@
     <string name="failed_edit_comment">Comment edit failed. Try again.</string>
     <string name="debug">Debug</string>
     <string name="view_comments">View Comments</string>
+    <string name="reverting">Reverting...</string>
```

Comment:
The `...` should be an ellipsis character (`…`). You can find one up above on `Completing…`.